src/CMakeLists.txt

```diff
@@ -37,11 +37,20 @@ endif()
 # Expose public API
 target_include_directories(benchmark PUBLIC ${PROJECT_SOURCE_DIR}/include)
 
+# Pick installation dir depending on bitness if not specified
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
```

Comment:
i don't know if i agree with my original assessment. it's not that important - they both work.